### PR TITLE
WIP: Add README.md

### PR DESCRIPTION
**Pull request by oharan2**
WIP untill #1 is merged for track code changes and wait for pre-commit config.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,109 @@
 # openshift-cli-installer
 Basic openshift install cli wrapper
+
```
Comment (myakove) [inline README.md:3]: Please add limitation section that state that we only support IPI install on AWS for now

Comment (myakove) [inline README.md:3]: Please add markdown link to pre-commit

**Review by myakove — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -1,2 +1,109 @@
...
+Pass `--clusters-install-data-directory` or set `CLUSTER_INSTALL_DATA_DIRECTORY` env var to indicate path for clusters installation data gathered
+
+Pass `--pull-secret-file` or set `PULL_SECRET` env var to indicate secret json file, which can also be obtained from console.redhat.com"
+
+Pass `--s3-bucket-name` and/or `--s3-bucket-path` to give S3 bucket name and path to store install data backups
+
+Flag `--cluster` accepts multiple args, the format is `arg=value;`
+
+###### Required args:
+* `name=name`: Name of the cluster to install/uninstall
```
Comment (omrirh) [inline README.md:53]:
```suggestion
* `name`: Name of the cluster to install/uninstall
```

File: README.md [hunk 3]
```diff
@@ -1,2 +1,109 @@
...
+```
+
+Each command can be run via container `podman run quay.io/redhat_msi/openshift-cli-installer` or via poetry command `poetry run app/cli.py`
+
+## Clusters
+
+Cluster/s to create or destroy.
+Flag --action accepts either "create" or "destroy"
+
+### User args
```
Comment (omrirh) [inline README.md:40]: maybe we should specify the defaults for variables?

**Pull request closed without merging**